### PR TITLE
v1.0.3

### PR DESCRIPTION
**Pull request by mattbarker016**

### CODE REVIEW

File: TCAT/Model/Route.swift
Comment (AAAstorga) [inline TCAT/Model/Route.swift:128]: good work here

**Review by AAAstorga — APPROVED** [review]
I approve if you don't need to change anything else. Please add the issues you solved to this PR though, so they close out when you merge the PR. If you make significant changes, please wait until morning for another review. Last thing we want is a broken, rushed version of Ithaca transit. Thanks for the quick action on these issues.

**Comment by mattbarker016** [comment]
Resolves Issues #124 #125 #131.

**Comment by mattbarker016** [comment]
Very tempted to merge and ship, but will wait. @AAAstorga approve ASAP and I'll get this thing moving through.

**Review by AAAstorga — APPROVED** [review]
Thanks for the quick fixes. If it all works on client then you are good! Get this sucker uploaded and released!

**Pull request merged**
